@rovere , HLT timing test still fails with error 
```
+ cmsDriver.py Phase2 -s L1P2GT,HLT:75e33_timing --processName=HLTX --conditions auto:phase2_realistic_T33 --geometry Extended2026D110 --era Phase2C17I13M9 --customise SLHCUpgradeSimulations/Configuration/aging.customise_aging_1000 --eventcontent FEVTDEBUGHLT --filein= --mc --nThreads 4 '--inputCommands=keep *, drop *_hlt*_*_HLT, drop triggerTriggerFilterObjectWithRefs_l1t*_*_HLT' -n 1000 --no_exec '--output={}'
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02850/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-14-2300/bin/el8_amd64_gcc12/cmsDriver.py", line 40, in <module>
    run()
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02850/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-14-2300/bin/el8_amd64_gcc12/cmsDriver.py", line 11, in run
    options = OptionsFromCommandLine()
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02850/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_X_2024-08-15-1100/src/Configuration/Applications/python/cmsDriverOptions.py", line 36, in OptionsFromCommandLine
    options=OptionsFromItems(sys.argv[1:])
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02850/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_X_2024-08-15-1100/src/Configuration/Applications/python/cmsDriverOptions.py", line 153, in OptionsFromItems
    options.filein=trimmedEvtType+"_"+prec_step[first_step]+"."+filesuffix
KeyError: 'L1P2GT'

```

--- a/HLTrigger/Configuration/python/HLT_75e33/test/runHLTTiming.sh
+++ b/HLTrigger/Configuration/python/HLT_75e33/test/runHLTTiming.sh
@@ -7,7 +7,7 @@ set -e
 set -o pipefail
 
 FOLDER_FILES="/data/user/${USER}/"
-DATASET="/RelValTTbar_14TeV/CMSSW_14_1_0_pre5-PU_140X_mcRun4_realistic_v4_STD_2026D110_PU-v2/GEN-SIM-DIGI-RAW"
+DATASET="/RelValTTbar_14TeV/CMSSW_14_1_0_pre6-PU_141X_mcRun4_realistic_v1_STD_2026D110_PU-v3/GEN-SIM-DIGI-RAW"
 FILES=( $(dasgoclient -query="file dataset=${DATASET}" --limit=-1 | sort | head -4) )
 
 for f in ${FILES[@]}; do

--- a/HLTrigger/Configuration/python/HLT_75e33/test/runHLTTiming.sh
+++ b/HLTrigger/Configuration/python/HLT_75e33/test/runHLTTiming.sh
@@ -7,7 +7,7 @@ set -e
 set -o pipefail
 
 FOLDER_FILES="/data/user/${USER}/"
-DATASET="/RelValTTbar_14TeV/CMSSW_14_1_0_pre1-PU_140X_mcRun4_realistic_v2_STD_2026D98_PU-v1/GEN-SIM-DIGI-RAW"
+DATASET="/RelValTTbar_14TeV/CMSSW_14_1_0_pre5-PU_140X_mcRun4_realistic_v4_STD_2026D110_PU-v2/GEN-SIM-DIGI-RAW"
 FILES=( $(dasgoclient -query="file dataset=${DATASET}" --limit=-1 | sort | head -4) )
 
 for f in ${FILES[@]}; do
@@ -37,7 +37,7 @@ ALL_FILES="${ALL_FILES%?}"
 echo "Discovered files: $ALL_FILES"
 
 cmsDriver.py Phase2 -s L1P2GT,HLT:75e33_timing --processName=HLTX \
-  --conditions auto:phase2_realistic_T25 --geometry Extended2026D98 \
+  --conditions auto:phase2_realistic_T33 --geometry Extended2026D110 \
   --era Phase2C17I13M9 \
   --customise SLHCUpgradeSimulations/Configuration/aging.customise_aging_1000 \
   --eventcontent FEVTDEBUGHLT \

